src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/InputUnusedImportFromJavaLang.java

```diff
@@ -0,0 +1,44 @@
+package com.puppycrawl.tools.checkstyle.checks.imports;
+
+import java.lang.String; // VIOLATION
+import java.lang.Math; // VIOLATION
+import java.lang.Class; // VIOLATION
+import java.lang.Exception; // VIOLATION
+import java.lang.Runnable; // VIOLATION
+import java.lang.RuntimeException; // VIOLATION
+import java.lang.ProcessBuilder; // VIOLATION
+import java.lang.Double; // VIOLATION
+import java.lang.Integer; // VIOLATION
+import java.lang.Float; // VIOLATION
+import java.lang.Short; // VIOLATION
+
+import java.lang.annotation.Annotation; // OK
+import java.lang.reflect.Modifier; // OK
+import java.lang.reflect.Field; // OK
+
+import java.lang.*; // OK
```

Comment:
Why isn't this a violation? Isn't it importing everything in `java.lang` and nothing in its sub-packages?
`import java.lang.*;` doesn't import `java.lang.reflect.Field` right? I tested this out and the compiler couldn't resolve `Field`.
This star import should be no different than importing a lang class like `Short`.